### ISSUE TITLE
balena preload fails with version >12.37.0 and openBalena 3.2.1

### ISSUE THREAD

**Issue by rickardgranberg** [balena-io/balena-cli#2231]
# Expected Behavior

```
balena preload target.img --app <appname> --commit <hash>

<log output reduced for brevity>
...
Pulling n images [========================] 100%
```

# Actual Behavior

The above command works fine using `v 12.36.1`, but any version above that fails:
```
balena preload target.img --app <appname> --commit <hash>

Building Docker preloader image. [                        ] 0%
Building Docker preloader image. [===                     ] 12%
Building Docker preloader image. [======                  ] 25%
Building Docker preloader image. [=========               ] 37%
Building Docker preloader image. [============            ] 50%
Building Docker preloader image. [===============         ] 62%
Building Docker preloader image. [==================      ] 75%
Building Docker preloader image. [=====================   ] 87%
Building Docker preloader image. [========================] 100%




Error: getaddrinfo ENOTFOUND registry2<our-openbalena-domain>
```

As I understand it, `registry2...` is a balenaCloud enpoint that openBalena does not have, but why is it trying to reach that endpoint? 
`balena deploy` works fine btw.

Theres a forum thread re: this problem as well: https://forums.balena.io/t/balena-preload-points-to-wrong-registry/260600

# Steps to Reproduce the Problem

1.Upgrade balena cli to 12.37.0 or above
1.Run balena preload

# Specifications

- **balena CLI version:** > 12.37.0 
- **Cloud backend: openBalena 3.2.1
- **Operating system version:** Ubuntu 18.04
- **32/64 bit OS and processor:** 64-bit Linux on 64-bit Intel processor
- **Install method:** zip package
- **If npm install, Node.js and npm version:** 12.18.1

# Additional References

**Comment by pdcastro** [comment]
Connects-to: balena-io-modules/balena-preload/issues/236

**Comment by maggie0002** [comment]
Someone provided me a temporary fix for this over on the forums: https://forums.balena.io/t/balena-preload-points-to-wrong-registry/260600/35?u=maggie

**Comment by pdcastro** [comment]
This issue should be fixed in CLI release v12.44.16 or later, :tada:   thanks to @klutchell 's PRs balena-io-modules/balena-preload/pull/244 and #2269 💯
